dashboard/test/ui/features/support/dashboard_helpers.rb

```diff
@@ -9,6 +9,21 @@ def require_rails_env
     puts "Requiring rails env took #{finish - start} seconds"
     @rails_loaded = true
   end
+
+  # Sets DCDO for a test scenario.
+  # @see Rack::CookieDCDO
+  def set_dcdo(key, value)
```

Comment:
nit; can this be given a more explicit name? I'd like it to be made clear to someone reading the code that invokes this method that it is specifically setting a cookie which will override the actual value set in DCDO just for testing purposes, rather than actually setting a value on the DCDO store itself.